### ISSUE TITLE
Release 5.4.0 not compatible with Python 3.7

### ISSUE THREAD

**Issue by ericvanbuhler** [miguelgrinberg/Flask-SocketIO#2096]
**Describe the bug**
The 5.4.0 release fails to import in Python 3.7 due to an unsupported f-string expression. This line may be the issue: https://github.com/miguelgrinberg/flask-socketio/commit/45990f6721004d97fbac5b02edcd119c515155d6#diff-d4032ca32e5dce64b1e46683f4bc8544ccefadb6e8f3e09a62152dda9c0a0578R293.

The import error looks like this:
```python
>>> import flask_socketio
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "<fstring>", line 1
    (real_msg=)
             ^
SyntaxError: invalid syntax
```

Support in Python is described here: https://docs.python.org/3/whatsnew/3.8.html#f-strings-support-for-self-documenting-expressions-and-debugging 

**To Reproduce**
1. Create and activate Python virtualenv with Python 3.7
2. Install flask_socketio 5.4.0 (`pip install flask_socketio==5.4.0`)
3. Within the Python virtualenv, import flask_socketio to see the import error shown above (`import flask_socketio`)

By doing the same steps in a Python environment of python 3.8+, the import will succeed. Additionally, release 5.3.7 can be successfully imported in Python 3.7.

**Expected behavior**
I expected the import to succeed for Python 3.7.

**Logs**
Since it doesn't get past import, I don't think it will generate any logs.

**Comment by miguelgrinberg** [comment]
I have release 5.4.1 removing this print statement that I left over by mistake.

In any case, note that Python 3.7 isn't officially supported anymore by this package, so consider upgrading your Python because even though I'm not actively trying to break Python 3.7 I'm not testing that version anymore and when things break for a good reason I'm not going to accept fixes that are just for 3.7.

**Comment by ericvanbuhler** [comment]
Thank you for the quick fix, @miguelgrinberg! The `pyproject.toml` file still specifies support for Python >= 3.6 so I assumed support for 3.7. I'm looking at dropping 3.7 support on my side soon so it won't be a long-term issue.

**Comment by miguelgrinberg** [comment]
I don't actively try to stay compatible with 3.7 and older. I keep the `requires` lower because there is always people who want to use older versions, but I don't test nor intend to apply fixes for anything below 3.8, so it is a "use at your own risk" kind of deal.
